src/tests/locationChooser.test.ts

```diff
@@ -0,0 +1,763 @@
+import { expect, test } from '@/fixtures/fixtures';
+import LocationChooser from '@/pages/LocationChooser';
+import LoginPage from '@/pages/LoginPage';
+import Navbar from '@/pages/Navbar';
+import AppConfig from '@/utils/AppConfig';
+
+//tests are covering all steps from test case OBPIH-4644 Location Chooser
+test.describe('Check if depot location is present in location chooser', () => {
+  const ORGANIZATION_NAME = 'E2E-test-Organization';
+  const GROUP_NAME = 'E2E-test-Group';
+  const LOCATION_NAME = 'E2E-test-Depot';
+  const LOCATION_COLOR = '#009dd1';
+
+  test.beforeAll(
+    async ({
+      page,
+      navbar,
+      organizationListPage,
+      createOrganizationPage,
+      editOrganizationPage,
+      locationGroupsListPage,
+      createLocationGroupPage,
+      locationListPage,
+      createLocationPage,
+    }) => {
+      await test.step('Go to create organization page', async () => {
+        await page.goto('./dashboard');
+        await navbar.configurationButton.click();
+        await navbar.getNavItem('Organizations').click();
+        await organizationListPage.createOrganizationButton.click();
+      });
```

Comment:
I noticed this in many places so this is more of a question what we want to test.

There are many places where we go to Dashboard page then use the navbar to go to certain page, fi we don;t care about using the navbar all that much we can shorten the test by going directly to organization create page.
We can implement following function in `CreateOrganizationPage.ts`

```ts
async goToPage() {
  await this.page.goto('./organization/create');
}
```

and then in the test just redirect to the create page
```ts
  await organizationListPage.goToPage();
```
what do you think?